README.md

```diff
@@ -50,6 +57,7 @@ Where:
 * `APPLICATION_ID` is the identifier generated by Yoti Dashboard when you create your app.
 * `path/to/your-application-pem-file.pem` is the path to the pem file your browser generates for you, when you create your app on Yoti Dashboard.
 
+See node/example/simple-login/index.js for example version support.
```

Comment:
I don't understand the meaning of this sentence. Versions are not mentioned in this section. You should make the path a link :)